integration-tests/cli/helpers/utils.js

```diff
@@ -40,7 +40,7 @@ export function clearData(apiConfig, entityName) {
   })
 }
 
-export function createData(apiConfig, entityName, data) {
+export function createData(apiConfig, entityName, data, id) {
```

Comment:
Nitpick: I suggest to use named arguments instead of positional arguments. Makes it easier to add/remove arguments.

```js
createData({ apiConfig, entityName, data, id })
```

Reply:
If this is used across the library I don't see it being an issue for this PR.

Reply:
Of course, it's more of a general thing (hence the nitpick)

Reply:
Good idea. Will keep in mind for future PRs :-)